settings.yml

```
@@ -80,7 +80,7 @@ variables:
   ##   Connection settings   ##
   #############################
 
-  serviceConnection: 'CARML-CSU-Tenant-Connection'
+  serviceConnection: 'skhan-bicep-serviceConnection'
```

Please revert changes to this file
```suggestion
  serviceConnection: 'CARML-CSU-Tenant-Connection'
```